INPUT_SCHEMA.json

```diff
@@ -52,6 +52,18 @@
       "description": "If checked results includes reviews data",
       "default": false
     },
+    "maxReviews": {
+      "title": "Limit reviews for each place",
+      "type": "number",
```

Comment:
need to set the `editor` here and for `maxItems` to `number` (and the type being `integer`)